recipe/meta.yaml

```diff
@@ -11,18 +11,28 @@ source:
   sha256: 49b9deac842b3698fc9011b7678f943b4e36136e2ab711f535c1961ff2746e14
 
 build:
-  number: 0
-  skip: true  # [win]
+  number: 1
+  ignore_run_exports:
+    - libgcc-ng
+    - libstdcxx-ng
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - cmake
-    - ninja
+    - cmake {{ cmake }}
```

Comment:
Does this need `make` added as well?

Reply:
Ideally it should need make as well. But it didn't complain either on ppc or x86. I'll recheck.